src/schemas/secrets.schema.json

```diff
@@ -73,6 +73,11 @@
       "type":"boolean",
       "description": "Sanitize the HTML output to guard against XSS attacks. \n\n**Note:** this flag applies a pretty aggressive DOM filtering that will strip out a lot of HTML that your authors might find useful. The setting is meant for processing truly untrusted inputs, such as comments in a social media site.",
       "default":false
+    },
+    "API_RESOLVE_GIT_REF": {
```

Comment:
@trieloff should we name this `RESOLVE_GITREF_SERVICE` analog to the `EMBED_SERVICE` ?

Reply:
Good idea.